pandera/schemas.py

```diff
@@ -165,7 +166,7 @@ def __init__(
         self.index = index
         self.strict = strict
         self.name = name
-        self._pandas_dtype = pandas_dtype
+        self.dtype = dtype
```

Comment:
did you mean `self._dtype = dtype`?

Reply:
Nope, this is intended. It will call the setter property that will convert to a engine-specific dtype.

Reply:
cool